ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AVD_LAB.yml

```diff
@@ -30,9 +30,12 @@ snmp_settings:
 mgmt_gateway: 192.168.200.5
 
 # dns servers.
-name_servers:
-  - 192.168.200.5
-  - 8.8.8.8
+dns_settings:
```

Comment:
Can we test also the extra VRF and in particular default?
also probably need to test the `use_inband_interface_vrf` somewhere

Reply:
Yes I forgot to add a unit test for all variations. I will do that.